src/components/alarm/AlarmController.cpp

```diff
@@ -48,10 +47,7 @@ void AlarmController::SetAlarmTime(uint8_t alarmHr, uint8_t alarmMin) {
   minutes = alarmMin;
 }
 
-void AlarmController::ScheduleAlarm() {
-  // Determine the next time the alarm needs to go off and set the app_timer
-  app_timer_stop(alarmAppTimer);
-
+std::chrono::time_point<std::chrono::system_clock, std::chrono::nanoseconds> AlarmController::TimePoint() {
```

Comment:
This returns the timepoint of the alarm right? Maybe a name like `CalculateAlarmTimePoint()` would be more descriptive.

Reply:
Fixed